github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: ci
+on: push
+
+jobs:
+  build_and_deploy:
+    env:
+      BASE_VERSION: 0.1
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Build
+      run: |
+        dotnet build STUN.sln -p:Configuration=Release
```

Comment:
I think build phase should build in debug mode, and package phase in release mode